overview/mkr.md

```diff
@@ -0,0 +1,25 @@
+{% hint style="warning" %} This documentation describes certain core aspects of MakerDAO's protocol that could change over time. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+# MKR Token
+
+MKR is an ERC-20 token native to the Ethereum Blockchain and serves as the governance token within the MakerDAO protocol, granting holders voting rights to actively shape decisions and uphold system stability. 
+
+The MKR token embodies MakerDAO's decentralization, empowering token holders with decision-making authority, nurturing an inclusive governance system. Active participation in governance decisions and ensuring protocol stability are pivotal roles that MKR token holders play in upholding MakerDAO's integrity and effectiveness.
+
+The MKR token plays a vital role in shaping the future of the decentralized ecosystem.
+
+## Purpose of MKR
+
+MKR token holders have the power to influence key decisions and shape the direction of the MakerDAO ecosystem. Through the MKR token, participants have the ability to actively engage in the governance process and vote on crucial protocol upgrades, risk parameters, and other important matters that affect the system and its growth.
```

Comment:
Feels repetitive here. We could combine this with the previous section. Again MKR has the following roles
1. Govern the protocol
2. Serve as a means to realize protocol profits
3. Cover Dai holders in case the protocol makes losses.